sdk/monitor/opentelemetry-exporter-azure-monitor/src/utils/spanUtils.ts

```diff
@@ -35,10 +38,26 @@ import { RemoteDependencyData, RequestData, TelemetryItem as Envelope } from "..
 function createTagsFromSpan(span: ReadableSpan): Tags {
   const context = getInstance();
   const tags: Tags = { ...context.tags };
+
   tags[AI_OPERATION_ID] = span.spanContext.traceId;
   if (span.parentSpanId) {
     tags[AI_OPERATION_PARENT_ID] = span.parentSpanId;
   }
+  if (span.resource && span.resource.labels) {
+    const serviceName = span.resource.labels[SERVICE_RESOURCE.NAME];
+    const serviceNamespace = span.resource.labels[SERVICE_RESOURCE.NAMESPACE];
+    if (serviceName) {
+      if (serviceNamespace) {
+        tags[AI_CLOUD_ROLE] = serviceNamespace + "." + serviceName;
+      } else {
+        tags[AI_CLOUD_ROLE] = serviceName.toString();
+      }
+    }
+    if (span.resource.labels[SERVICE_RESOURCE.INSTANCE_ID]) {
+      tags[AI_CLOUD_ROLE_INSTACE] = span.resource.labels[SERVICE_RESOURCE.INSTANCE_ID].toString();
```

Comment:
do you ever have to worry about `span.resource.labels[SERVICE_RESOURCE.INSTANCE_ID]` being undefined? One option is to use `String(span.resource.labels[SERVICE_RESOURCE.INSTANCE_ID])` instead of relying on `toString()` to be present